examples/ios/swift/ListSwiftUI/Models/Ingredient.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import RealmSwift
+
+final class Ingredient: Object {
+    @objc dynamic var name: String? = nil
+    let recipes = LinkingObjects(fromType: Recipe.self,
+                                 property: "ingredients")
+    @objc dynamic private var _foodType: String = "food"
+
+    var foodType: FoodType? {
```

Comment:
This would be a good place for a comment explaining why there's this wrapper, as part of the point of a sample is to educate users who are trying to learn how to use Realm. In general anything not 100% obvious related to Realm in a sample should have a comment explaining what it's doing targeted at someone who is just getting started with Realm.